microservices/api/src/controllers/ConfigurationController.php

```diff
@@ -21,9 +20,18 @@ public function actionCountries(
 
     public function actionLocales(
         ConfigurationProvider $configurationProvider,
-        Response $response
-    ): array|string {
+        Response              $response
+    ): array|string
+    {
         $response->headers->add('Cache-Control', 'max-age=604800,public');
         return $configurationProvider->getPlatformLocales();
     }
+
+    public function actionVisibilities(
+        ConfigurationProvider $configurationProvider,
+        Response              $response): array
```

Comment:
Don't do manual indenting like this, we use a tool for this : vendor/bin/ecs -- easy coding standard.